config/_default/params.toml

```diff
@@ -8,6 +8,9 @@
 colorScheme = "blowfish"
 defaultAppearance = "light" # valid options: light or dark
 autoSwitchAppearance = true
+# expose theme colors for iOS notch coloring (barring better options)
+iosLightOverride = "ffffff"
```

Comment:
delete these, not needed - should use the selected theme color set for the background.